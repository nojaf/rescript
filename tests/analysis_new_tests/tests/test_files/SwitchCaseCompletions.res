// == TEST: Empty case, array
let someStringArr = ["hello"]

let x = switch someStringArr {
  | 
//  ^com
}

// == TEST: Empty case, record
let x = switch TestTypeDefs.nestedTestRecord {
  | 
//  ^com
}

// == TEST: Empty case, bool
let x = switch true {
  | 
//  ^com
}
