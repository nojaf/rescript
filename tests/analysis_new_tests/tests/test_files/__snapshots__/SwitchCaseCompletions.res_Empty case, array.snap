Complete /Users/nojaf/Projects/rescript/tests/analysis_new_tests/tests/test_files/.build/SwitchCaseCompletions_1.res 4:4
Found Completable: Cpattern

1 + // Empty case, array
2 + let someStringArr = ["hello"]
3 + 
4 + let x = switch someStringArr {
                   -------------  
5 +   | 
6 + //  ^com
7 + }

[{
    "label": "[]",
    "kind": 12,
    "tags": [],
    "detail": "array<string>",
    "documentation": null,
    "sortText": "A",
    "insertText": "[$0]",
    "insertTextFormat": 2
  }]

